superset-frontend/spec/javascripts/dashboard/components/FilterTooltipWrapper_spec.jsx

```diff
@@ -19,6 +19,7 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 import { Overlay, Tooltip } from 'react-bootstrap';
+import { promiseTimeout } from '@superset-ui/core';
```

Comment:
Probably not needed if you are not using it.

Reply:
ugh, yup. idk why a linter didn't catch this :(